source/Calamari.Testing/CommandTestBuilder.cs

```diff
@@ -263,20 +262,20 @@ async Task<TestCalamariCommandResult> ExecuteActionHandler(List<string> args)
 
                     CopyFilesToWorkingFolder(workingPath);
 
-                    result = await ExecuteActionHandler(args);
+                    result = await ExecuteActionHandler(args, workingPath);
+
+                    if (assertWasSuccess)
+                    {
+                        result.WasSuccessful.Should().BeTrue($"{command} execute result was unsuccessful");
```

Comment:
Guessing this moved because you still needed the current directory ?

Reply:
yes, so we can assert before the directory is deleted